### PR TITLE
Select component tagTemplate

### PR DESCRIPTION
**Pull request by itsmarkodowd**
#### Checklist
* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licenced under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-3471

#### Description of Proposed Changes

- added tagTemplate input to select component
- forwarded tagTemplate to tag-input component
- updated documentation to include tagTemplate in select section

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_

### CODE REVIEW

**Comment by donna-marijne** [comment]
Thanks! Sorry about that. Will review soon(TM).

File: docs/app/pages/components/components-sections/select/select/select.component.html [hunk 1]
```diff
@@ -206,6 +206,22 @@
     <tr uxd-api-property name="placeholder" type="string">
         The placeholder text which appears in the text input area when it is empty.
     </tr>
+    <tr uxd-api-property name="tagTemplate" type="TemplateRef">
+        <p>A template which will be rendered as the content of each tag. The following context
+            properties are available in the template:</p>
```
Comment (donna-marijne) [inline docs/app/pages/components/components-sections/select/select/select.component.html]: I understand that this is lifted from the Tag Input documentation, but I think it should be amended to be more relevant to the select component. The "tag" in this case is a selected option in multiple select mode, so therefore "selected option" may be a better substitution for "tag" throughout this documentation entry.

File: docs/app/pages/components/components-sections/select/select/select.component.html [hunk 2]
```diff
@@ -206,6 +206,22 @@
...
+        <ul>
+            <li><code>tag: any</code> - the string or custom object representing the tag.</li>
+            <li><code>index: number</code> - the zero-based index of the tag as it appears in the tag
+            input.</li>
+            <li><code>api: TagApi</code> - provides the functions <code>getTagDisplay</code>,
+            <code>removeTagAt</code> and <code>canRemoveTagAt</code>. See the section below for more
+            information.</li>
```
Comment (donna-marijne) [inline docs/app/pages/components/components-sections/select/select/select.component.html]: There is no "section below" that is relevant here.

**Comment by TaraTodd123** [comment]
Closing this pull request for an updated branch with additional changes

**Pull request closed without merging**